index.html

```diff
@@ -10,7 +10,7 @@
                                       Bootstrap, JavaScript, jQuery, Reactjs.  Experts in responsive Web Design.  Strong experience with Cross Browser Compatibility issues and Optimization for web">
     <link rel="stylesheet" href="style.css">
   </head>
-  <body>
+  <body id="body">
```

Comment:
- [ ] [Optional] Is not necessary to create an id property for body tag, since this one is unique for each document. You can directly call it by its name

Reply:
thank you for the knowledge you have shared. I will follow your suggestion.